h2o-genmodel/src/main/java/hex/genmodel/attributes/parameters/KeyValue.java

```diff
@@ -0,0 +1,21 @@
+package hex.genmodel.attributes.parameters;
+
+public class KeyValue {
```

Comment:
Can't we use `water.Iced.KeyValue` instead or is there any limitation?

Reply:
This is MOJO